ci/build_wheel_python.sh

```diff
@@ -29,3 +29,7 @@ mkdir -p final_dist
 python -m auditwheel repair -w final_dist dist/*
 
 RAPIDS_PY_WHEEL_NAME="${package_name}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 python final_dist
+
+# switch back to the root of the repo and check symbol visibility
+cd ../../
```

Comment:
Rather than do a `cd`, we should use `pushd $package_dir` above and then popd so that you don't have to recall the relative path here.

Reply:
good idea, did that in https://github.com/rapidsai/rmm/pull/1644/commits/8d5d060c061853663908d2fab1628fd326497229